specification/playwrighttesting/resource-manager/readme.md

```diff
@@ -32,6 +32,11 @@ openapi-subtype: rpaas
 tag: package-2023-10-01-preview
 ```
 
+``` yaml
```

Comment:
why we don't need to update `client-flattened-annotation-target: disabled` in readme.java.md for this RP?

Reply:
It is added by @weidongxu-microsoft. And this PR is only for Python/JS/C#

Reply:
already in main https://github.com/Azure/azure-rest-api-specs/blob/main/specification/playwrighttesting/resource-manager/readme.java.md